### ISSUE TITLE
Error during Publish Command

### ISSUE THREAD

**Issue by EJ-RG** [autodesk-platform-services/aps-hubs-bulk-files-manager#2]
I am very new to APS and even Git..
But going through the installation steps - I can't find the "Bulk-File-Manager.exe" file.

From executing the second command, "dotnet publish -r win-x64" in returned following message.
--------------------------------------------------------------------------------------------------------------- 
PS C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files-manager> dotnet publish -r win-x64
  Determining projects to restore...
  All projects are up-to-date for restore.
  Ac.Net.Authentication -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-
  bulk-files-manager\Ac.Net.Authentication\bin\Release\netstandard2.1\win-x64\Ac.Net.Authentication.dll     
  Ac.Net.Authentication -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-
  bulk-files-manager\Ac.Net.Authentication\bin\Release\netstandard2.1\win-x64\publish\
  PluginBase -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files- 
  manager\PluginBase\bin\Release\netstandard2.1\win-x64\PluginBase.dll
  PluginBase -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files-
  manager\PluginBase\bin\Release\netstandard2.1\win-x64\publish\
  Ac.Net.Authentication -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-
  bulk-files-manager\Ac.Net.Authentication\bin\Release\netstandard2.1\Ac.Net.Authentication.dll
  PluginBase -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files- 
  manager\PluginBase\bin\Release\netstandard2.1\PluginBase.dll
  ApsSettings.Data -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-
  files-manager\ApsSettings.Data\bin\Release\net8.0\ApsSettings.Data.dll
  ApsSettings.Data -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-
  files-manager\ApsSettings.Data\bin\Release\net8.0\win-x64\ApsSettings.Data.dll
  ApsSettings.Data -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-
  files-manager\ApsSettings.Data\bin\Release\net8.0\win-x64\publish\
  TestPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files-
  manager\TestPlugin\bin\Release\net8.0\win-x64\TestPlugin.dll
  TestPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files-
  manager\TestPlugin\bin\Release\net8.0\win-x64\publish\
  TestPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-files-
  manager\TestPlugin\bin\Release\net8.0\TestPlugin.dll
  MigrationPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-f 
  iles-manager\MigrationPlugin\bin\Release\net8.0\MigrationPlugin.dll
  MigrationPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-f 
  iles-manager\MigrationPlugin\bin\Release\net8.0\win-x64\MigrationPlugin.dll
  MigrationPlugin -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk-f
  iles-manager\MigrationPlugin\bin\Release\net8.0\win-x64\publish\
  Bulk-File-Manager -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk
  -files-manager\Bulk-Uploader-FrontEnd\bin\Release\net8.0-windows\win-x64\Bulk-File-Manager.dl

  added 314 packages, and audited 315 packages in 1m

  66 packages are looking for funding
    run `npm fund` for details

  3 high severity vulnerabilities

  To address issues that do not require attention, run:
    npm audit fix

  Some issues need review, and may require choosing
  a different dependency.

  Run `npm audit` for details.
  
  > bulk-file-manager@0.0.0 build
  > tsc && vite build

(!) Some chunks are larger than 500 kBs after minification. Consider:
  - Using dynamic import() to code-split the application
  - Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-opt
  ions/#output-manualchunks
  - Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
  Ô£ô built in 25.24s
  Bulk-File-Manager -> C:\Users\chonge\T\Documents\GitHub\ACC-BIM-REVIT\aps-hubs-bulk
  -files-manager\Bulk-Uploader-FrontEnd\bin\Release\net8.0-windows\win-x64\publish\

**Comment by EJ-RG** [comment]
Found the .exe file

But it shows the attached error when running the .exe file.
![image](https://github.com/user-attachments/assets/8e4cd9fd-fb50-4778-bd56-67ef7975d78f)

**Comment by auredor** [comment]
Hi @EJ-RG, I'm facing the same issue (HTTP ERROR 500). For some reasons, the application does work at first start but then I got the error if I try to start it a second times (even after computer restart). I've deleted all the files from the "Release" folder but it doesn't help. The only way to troubleshoot the issue is to download a new github version on the computer. So, it looks like there are some files (or data in a db ???) that prevent the proper behavior. Have you succeed to identify them ? Thanks.
